frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.jsx

```diff
@@ -16,7 +18,7 @@ export default class ExpressionWidget extends Component {
     query: PropTypes.object.isRequired,
     onChangeExpression: PropTypes.func.isRequired,
     onRemoveExpression: PropTypes.func,
-    onClose: PropTypes.func.isRequired,
+    onClose: PropTypes.func,
```

Comment:
Was getting a warning here when using it in dev. Hope it's ok to update the PropType along with the fix.